templates/rpm/project.spec.erb

```diff
@@ -175,6 +228,23 @@ done
 <%- end -%>
 
 %preun
+# Run pre-uninstall scripts on upgrade if defined
+<%- scripts = get_preremove_actions("upgrade") -%>
```

Comment:
That's the choice, comments that refer to nothing, or comments that refer to empty if statements. What's more obnoxious? Or, maybe, what's the least obnoxious?